src/import/sst_service.rs

```diff
@@ -543,8 +551,10 @@ macro_rules! impl_write {
         ) {
             let import = self.importer.clone();
             let tablets = self.tablets.clone();
-            let (rx, buf_driver) =
-                create_stream_with_buffer(stream, self.cfg.stream_channel_window);
+            let (rx, buf_driver) = create_stream_with_buffer(
+                stream,
+                self.cfg.0.as_ref().read().unwrap().stream_channel_window,
```

Comment:
You can use the `HandyRwLock` extension to replace the pattern `.read().unwrap()`.
```suggestion
                self.cfg.0.rl().stream_channel_window,
```

Reply:
I mean you can use `tikv_util::HandyRwLock` and then replace the `read().unwrap()` call chain with `rl()`.

https://github.com/tikv/tikv/blob/fd2db9a796b16d7665927340d9ebdc83022ea0e2/components/tikv_util/src/lib.rs#L175-L183